bin/node-template/node/src/service.rs

```diff
@@ -196,6 +197,8 @@ pub fn new_full(mut config: Configuration) -> Result<TaskManager, ServiceError>
 			task_manager: &mut task_manager,
 			transaction_pool: transaction_pool.clone(),
 			rpc_extensions_builder,
+			// TODO: (dp) implement
+			rpsee_builder: Box::new(|_, _| { RpcModule::new(()) }),
```

Comment:
we will take of it in another PR?

Reply:
Yeah, or, I'm not sure what `rpc_extensions_builder` is here, but in a few places it's just an empty closure just like I do here.